src/core/TimePos.cpp

```diff
@@ -161,11 +161,7 @@ tick_t TimePos::getTickWithinBeat( const TimeSig &sig ) const
 
 f_cnt_t TimePos::frames( const float framesPerTick ) const
 {
-	if( m_ticks >= 0 )
-	{
-		return static_cast<f_cnt_t>( m_ticks * framesPerTick );
-	}
-	return 0;
+	return static_cast<f_cnt_t>(m_ticks * framesPerTick);
```

Comment:
Have you checked for any potential issues with the removal of `m_ticks >= 0` check? I agree the check is not always desired, though.

Reply:
I had an assert there before, it wasn't triggered back then in any of the tests but I can add it again. Even if this PR doesn't leave any `m_ticks < 0` condition, who knows, another PR might introduce it again by accident, so I added it again!